application/features/user/user.feature

```diff
@@ -3,27 +3,38 @@ Feature: User
   As an authorized user
   I need to be able to manage user information
 
-  Scenario: Attempt to create a user as an unauthorized user
+  Scenario Outline: Attempt to act upon a user as an unauthorized user
```

Comment:
Do you think it would be worth using a different word for the user (record) and for the user (initiating the HTTP request)?